docs/src/main/sphinx/release/release-475.md

```diff
@@ -14,6 +14,7 @@
 * Fix incorrect results when using window functions with `DISTINCT`. ({issue}`25434`)
 * Fix query failures with `EXCEEDED_LOCAL_MEMORY_LIMIT` errors due to incorrect memory accounting. ({issue}`25600`)
 * Properly handle inline session properties for `EXPLAIN` queries. ({issue}`25496`)
+* Add support for exporting OpenTelemetry traces using HTTP protocol with the `tracing.exporter.protocol` configuration property (https://github.com/airlift/airlift/pull/1443).
```

Comment:
Please change the link to airlift bump PR in Trino.

Reply:
Specifically the PR is https://github.com/trinodb/trino/pull/25648 .. please confirm @wendigo and ensure that future airlift upgrade include necessary changes like this.

Reply:
The right PR number is https://github.com/trinodb/trino/pull/25573

Reply:
```suggestion
* Add support for exporting OpenTelemetry traces using the HTTP protocol with the `tracing.exporter.protocol` configuration property set to `http/protobuf`. ({issue`25648`)
```

Reply:
Also wrap the sentence

Reply:
oh.. thanks @ebyhr  .. adjusting